.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+<!-- If this PR adds a CEP, do not replicate information in the document. Give a brief summary and link to related issues and PRs. -->
+
+# Summary of Changes
+<!--- In one or more sentences, describe the PR you are submitting. -->
+
+
+# Related CEPs and Issues
+<!--- Reference any related CEPs or issues and describe how this PR contributes to or addresses the problem. -->
+
+
+# Associated Developers
+<!--- Please mention any developers who should be alerted of this PR. -->
+
+
+# Design Notes
+<!--- Describe any design decisions or trade-offs you made. -->
+
+
+# Testing and Validation
+<!--- Describe how this change was tested. -->
+
+
+# Checklist
+- [ ] Read the [Contributing to Cyclus](https://fuelcycle.org/kernel/contributing_to_cyclus.html) guide.
+- [ ] Compile and run locally.
+- [ ] Add or update tests.
+- [ ] Document if needed.
+- [ ] Follow style guidelines.
+- [ ] Update the changelog.
+- [ ] Address all review comments.
```

Comment:
[pedantic] I don't think we need periods at the end of these
```suggestion
- [ ] Read the [Contributing to Cyclus](https://fuelcycle.org/kernel/contributing_to_cyclus.html) guide
- [ ] Compile and run locally
- [ ] Add or update tests
- [ ] Document if needed
- [ ] Follow style guidelines
- [ ] Update the changelog
- [ ] Address all review comments
```

Reply:
I understand removing periods for a less formal vibe! In undergrad we were told to end list items with punctuation, and I've done that ever since. Both APA and MLA guides agree that complete imperatives should have a period, but the newer versions leave options incomplete items

Reply:
I'll accept that formally, these probably should have punctuation because they are imperitives and not lists of items (for which commas would be better), but it did look a little abrupt... (and I think that even though I'm Gen-X 🤣 )

Reply:
Great point!